ladybug/_datacollectionbase.py

```diff
@@ -586,6 +636,13 @@ def percentile_function(vals):
             return self._percentile(vals, percentile)
         return percentile_function
 
+    def _get_mutable_enumeration(self):
+        self._enumeration = _DataCollectionEnumeration(import_modules=False)
+
+    def _mutable_message(self):
```

Comment:
It works fine as it is but it might look cleaner to be set as a property. so you can get it as `self._mutable_message`.

Reply:
Also good idea.  Done!